packages/message-pickup-repository-pg/src/config/dbCollections.ts

```diff
@@ -0,0 +1,30 @@
+export const messagesTableName = 'queuedmessage'
+
+export const createTypeMessageState = `CREATE TYPE message_state AS ENUM ('pending', 'sending');`
+
+export const createTableMessage = `
+CREATE TABLE IF NOT EXISTS ${messagesTableName} (
+  id UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+  connectionId VARCHAR(255),
+  recipientDids TEXT[],
+  encryptedMessage JSONB,
```

Comment:
Not an SQL expert here, but I think we should use consistent naming for tables and fields. I see that most commonly they use snake_case conventions, so here it would be connection_id, recipient_dids, encrypted_message, etc.